process/datadog_checks/process/process.py

```diff
@@ -334,8 +336,7 @@ def file_to_string(path):
             self.log.debug('error getting proc stats: file_to_string failed',
                            'for /{}/{}/stat'.format(psutil.PROCFS_PATH, pid))
             return None
-
-        return map(lambda i: int(i), data.split()[9:13])
+        return (int(i) for i in data.split()[9:13])
```

Comment:
@nmuesch @zippolyte Just fyi this is a generator, not a tuple. The code doesn't do any indexing of this return value so this is still fine

Reply:
Ahh, thanks for the heads up 🙇